core/src/test/java/de/medavis/lct/core/patcher/ComponentMetaDataManagerTest.java

```diff
@@ -0,0 +1,75 @@
+/*-
+ * #%L
+ * License Compliance Tool - Implementation Core
+ * %%
+ * Copyright (C) 2022 - 2024 medavis GmbH
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package de.medavis.lct.core.patcher;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URISyntaxException;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+class ComponentMetaDataManagerTest {
+
+    @Mock()
+    private Logger mockedLogger;
+
+    @Test
+    void testFindMatch() throws URISyntaxException {
```

Comment:
This test tests code that is not used in production. Suggestion: Remove the tested method and the test.